framework/helpers/BaseStringHelper.php

```diff
@@ -527,4 +527,32 @@ public static function mask($string, $start, $length, $mask = '*') {
 
         return $masked;
     }
+
+    /**
+     * Returns the portion of the string that lies between the first occurrence of the start string
+     * and the last occurrence of the end string after that.
+     *
+     * @param string $string The input string.
+     * @param string $start The string marking the start of the portion to extract.
+     * @param string $end The string marking the end of the portion to extract.
+     * @return string The portion of the string between the first occurrence of start and the last occurrence of end.
+     */
+    public static function findBetween($string, $start, $end)
+    {
+        $startPos = mb_strpos($string, $start);
+
+        if ($startPos === false) {
+            return '';
```

Comment:
How about returning `null` when `start` or `end` cannot be found? It would allow to distinguish between `StringHelper::findBetween('FooBar', 'Foo', 'Bar')` and `StringHelper::findBetween('Baz', 'Foo', 'Bar')`

Reply:
Done